src/components/WalletModal/index.tsx

```diff
@@ -126,17 +125,17 @@ const sendAnalyticsEventAndUserInfo = (
   isReconnect: boolean
 ) => {
   sendAnalyticsEvent(EventName.WALLET_CONNECT_TXN_COMPLETED, {
-    result: WALLET_CONNECTION_RESULT.SUCCEEDED,
+    result: WalletConnectionResult.SUCCEEDED,
     wallet_address: account,
     wallet_type: walletType,
     is_reconnect: isReconnect,
   })
-  user.set(CUSTOM_USER_PROPERTIES.WALLET_ADDRESS, account)
-  user.set(CUSTOM_USER_PROPERTIES.WALLET_TYPE, walletType)
+  user.set(CustomerUserProperties.WALLET_ADDRESS, account)
```

Comment:
Why is this called `CustomerUserProperties`?

Reply:
Good catch, typo